src/components/UserManager.js

```diff
@@ -66,6 +67,7 @@ export const getUser = (callback) => {
                         group: group && !!group[0] ? group[0] : "",
                     };
                     api.setAccessToken(kc.token);
+                    updateSentryUser(user);
```

Comment:
it's not good place to set user from. Better to take it after user join room. So we can get publisher id and can search in janus.log.

Reply:
Or actually you can leave it here but add in clients to update user when it change

Reply:
Also, where do you clean it when he logout ? set it to null

Reply:
set user when you have user. Add janus session_id when you have it, videoroom handle id when you have it, remoteFeed when you have it. you get the point.

Also make sure you remove them when they go stale.

Maybe next PR if you wish

Reply:
Done.